README.md

```diff
@@ -8,13 +8,30 @@ To use the config:
 
 1. Install the repo:
    ```shell
-   npm install eslint-plugin-siren@sirensolutions/eslint-plugin-siren#<tag>
+   npm install eslint-plugin-siren@sirensolutions/eslint-plugin#<tag>
```

Comment:
This is wrong? We still have `-siren` before the `@`

Reply:
The [package name](https://github.com/sirensolutions/eslint-plugin/blob/master/package.json#L2) is `eslint-plugin-siren`, but the _repo_ is [`sirensolutions/eslint-plugin`](https://github.com/sirensolutions/eslint-plugin)

Reply:
:+1: